src/inputs/internal/ListBox.tsx

```diff
@@ -29,6 +30,7 @@ export function ListBox<O, V extends Key>(props: ListBoxProps<O, V>) {
     contrast = false,
     positionProps,
     loading,
+    disabledOptionsWithReasons = {},
```

Comment:
Fallback here felt cleaner rather than dealing with an actual undefined value downstream for the `ChipSelectField` component that uses the `ListBox` component, but doesn't support disabling options